scenario/actionhandler.go

```diff
@@ -461,16 +461,17 @@ func (act *Action) endAction(sessionState *session.State, actionState *action.St
 		containerActionEntry = originalActionEntry
 	}
 
-	if err := logResult(sessionState, actionState, actionState.Details, containerActionEntry); err != nil {
-		return err
-	}
+	logResultErr := logResult(sessionState, actionState, actionState.Details, containerActionEntry)
 	sessionState.LogEntry.LogDebugf("%s END", act.Type)
-
-	if err := logObjectRegressionData(sessionState); err != nil {
-		return err
+	logRegressionErr := logObjectRegressionData(sessionState)
+	var errs *multierror.Error
+	if logResultErr != nil {
```

Comment:
if `errs` is of type multi.Error, which it is here, you shouldn't need to check the sub errors for nil, when appending
from multierror.Append:
````golang
if e != nil {
	err.Errors = append(err.Errors, e)
}
```